meta/saimetadatatypes.h

```diff
@@ -190,11 +190,6 @@ typedef enum _sai_attr_value_type_t
     SAI_ATTR_VALUE_TYPE_INT32_RANGE,
 
     /**
-     * @brief Attribute value is vlan list.
-     */
-    SAI_ATTR_VALUE_TYPE_VLAN_LIST,
-
-    /**
```

Comment:
I think we should fix the lib as the enums should not be defined haphazardly. Keeping similar types organized together will help in the long run.